docs/release_process_quarterly.md

```diff
@@ -0,0 +1,467 @@
+    - [ ] Update git submodule references for the ‘jpo-ode’ project to point to tagged commits in projects with updated `master/main` branches.
+        - [ ] Open the jpo-ode project in an IDE.
+        - [ ] Navigate to the asn1_codec directory and run `git checkout tags/asn1_codec-x.x.x` to update the submodule reference.
+        - [ ] Navigate to the jpo-cvdp directory and run `git checkout tags/jpo-cvdp-x.x.x` to update the submodule reference.
+        - [ ] Navigate to the jpo-security-svcs directory and run `git checkout tags/jpo-security-svcs-x.x.x` to update the submodule reference.
+        - [ ] Navigate to the jpo-sdw-depositor directory and run `git checkout tags/jpo-sdw-depositor-x.x.x` to update the submodule reference.
+        - [ ] Navigate to the jpo-utils directory and run `git checkout tags/jpo-utils-x.x.x` to update the submodule reference. 
+        - [ ] Commit these changes to the `release/(year)-(quarter)` branch & push the changes to the remote repository.
+        - [ ] Ensure these changes pass CI/CD checks before continuing.
+    - [ ] Merge `release/(year)-(quarter)` branch into `master/main` branch for the jpo-ode project, and create a release with the version number of the release. (e.g. jpo-ode-x.x.x)
+    - [ ] Create docker image
+        - [ ] Use the release for the jpo-ode to produce docker image with the same tag name, containing the version number.
+        - [ ] Upload docker image to [DockerHub](https://hub.docker.com/u/usdotjpoode)
+        - [ ] Tag docker image with the version number of the app. (e.g. 1.0.0)
+        - [ ] Tag docker image with year and quarter of release. (e.g. 2024-Q2)
+        - [ ] Tag docker image with 'latest' tag for the most recent release.
+    - [ ] Merge master branch into develop branch & verify that CI/CD passes.
+
+### 4. DockerHub Image Testing
+    - [ ] image starts up correctly
+    - [ ] http endpoint is reachable
+    - [ ] tims can be successfully pushed to http endpoint
+    - [ ] capable of ingesting messages via udp (see scripts in `scripts/tests` directory)
+        - [ ] tims
+        - [ ] bsms
+        - [ ] ssms
+        - [ ] srms
+        - [ ] spats
+        - [ ] maps
+        - [ ] psms
+
+
+## jpo-geojsonconverter
+### Prerequisites
+    - [ ] jpo-ode release completed
+
+### 1. Code Ready & Release Notes
+    - [ ] Release notes drafted & added to `Release_notes.md` file in `docs` directory
+    - [ ] Code changes for release are merged into `develop`
+    - [ ] A new branch `release/(year)-(quarter)` is created from `develop`
+
+### 2. Preliminary Testing
+    - [ ] code compiles
+    - [ ] unit tests pass
+    - [ ] program starts up correctly
+    - [ ] program can be configured successfully
+    - [ ] MAP & SPaT messages are consumed successfully
+    - [ ] valid ProcessedMaps & ProcessedSpats are outputted
```

Comment:
Processed BSM messages are now produced by the GJC:

```suggestion
    - [ ] valid ProcessedMaps, ProcessedSpats, and ProcessedMaps are outputted
```

Reply:
Committed your suggestion!